metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -868,11 +868,14 @@
       ;; to send concept updates and deletions out of order.
       (if (and (util/is-tombstone? previous-revision) (nil? revision-id))
         previous-revision
-        (let [tombstone (merge previous-revision {:concept-id concept-id
+        (let [metadata (if (cu/generic-concept? concept-type)
+                         (:metadata previous-revision)
+                         "")
+              tombstone (merge previous-revision {:concept-id concept-id
                                                   :revision-id revision-id
                                                   :revision-date revision-date
                                                   :user-id user-id
-                                                  :metadata ""
+                                                  :metadata metadata
```

Comment:
Why do you need metadata for tombstoned concept?

Reply:
The generic documents have indexing information in configuration files.  In order to find the correct files, the MetadataSpecification section includes the schema name and version which corresponds to the configuration file that needs to be loaded and used.  Otherwise I won't be able to index the generic documents into the all revisions elastic index.